wellies/templates/suite.py_t

```diff
@@ -8,83 +10,111 @@ import wellies as wl
 ROOT_DIR = os.path.join(os.path.dirname(os.path.realpath(__file__)))
 
 
+@dataclass
+class EcflowServer:
+    """Class to store ecflow server configuration"""
+
+    hostname: str
+    user: str
+    deploy_dir: str
+
+
+class Host:
+    """Class to store host configuration"""
+    
+    def __init__(
+        self,
+        hostname: str,
+        user: str,
+        submit_arguments: dict,
+        job_out: str,
+        workdir: str,
+        troika_kwargs: Optional[dict] = None,
```

Comment:
I would maybe call this `pyflow_host_kwargs` ?? more verbose but a bit more general in my opinion